docs/generate-r-docs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cd ../api/r/cellxgene.census
+# Rscript -e 'install.packages("pkgdown")'
+# Rscript -e 'install.packages("usethis")'
+# Rscript -e 'usethis::use_pkgdown()'
```

Comment:
keep commented out code?

Reply:
I'll get rid of this for now - this might be useful if we transition this to build on GitHub pages using a workflow, which I still plan to do, but lower prio...